project/SPT.Custom/Patches/FixAirdropCrashPatch.cs

```
@@ -16,7 +18,8 @@ public class FixAirdropCrashPatch : ModulePatch
     {
         protected override MethodBase GetTargetMethod()
         {
-            return typeof(TarkovApplication).GetMethod(nameof(TarkovApplication.method_48));
+            return AccessTools.FirstMethod(typeof(TarkovApplication), (method) => 
+                method.GetParameters().Any(p => p.ParameterType == typeof(Result<ExitStatus, TimeSpan, MetricsClass>)));
```

Could we not stick to a more commonly used method and format in the code base?
I don't see this being used in any other places, you should do what e.g. [UnityWebRequestPatch](https://github.com/sp-tarkov/modules/blob/master/project/SPT.Core/Patches/UnityWebRequestPatch.cs) does instead.

---

If you want to use the `FirstMethod` approach, we tend to have the condition method as a standalone method, see this patch: https://github.com/sp-tarkov/modules/blob/4.0.x-dev/project/SPT.Custom/Patches/BetaLogoPatch.cs#L15

I would also recommend explicitly checking all parameters, or atleast the correct offset for this type, instead of checking for any parameter that has that type. We want the plugin to break if BSG does something unexpected, not silently fail like it's currently doing